docs/src/pages/faq.mdx

```diff
@@ -1,17 +1,17 @@
 ### The UploadThing components look all weird, what's going on?
 
-If your components appear unstyled, you likely forgot to import our CSS file
-from the React package. Don't forget to add the following to your root
-`layout.tsx` or `_app.tsx`:
+If the uploadThing components appear unstyled, you might have missed importing 
```

Comment:
casing - `uploadthing` is canonical. 

cc @t3dotgg we should probably do a pass to make sure all of the casing is consistent. I see that there is a number of places where we use `UploadThing` in the docs.

Reply:
I'll gladly make a PR when I have the time where I check all the mentions of uploadThing and change them to lowercase if necessary.